dask/layers.py

```diff
@@ -411,6 +411,22 @@ def __init__(
         self.name_input = name_input
         self.meta_input = meta_input
         self.parts_out = parts_out or range(npartitions)
+        self.split_name = "split-" + self.name
+
+        # Set high-priority to "split" tasks
```

Comment:
Maybe add some motivation like this: https://github.com/dask/dask/blob/0039002e12921520035462064f25f0da65ec36c6/dask/layers.py#L573-L583